readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to PR description

Reply:
https://Pavlo-lab-max-Kopalko.github.io/layout_stars/
https://Pavlo-lab-max-Kopalko.github.io/layout_stars/report/html_report/
Are we here now?